Test/datatypes/t4.bpl.expect

```diff
@@ -0,0 +1,2 @@
+
+Boogie program verifier finished with 3 verified, 0 errors
```

Comment:
It would be good to have some negative verification tests, too.

Reply:
Added t7.bpl.